packages/@o3r/pipeline/schematics/ng-add/templates/github/__dot__github/actions/setup/action.yml

```diff
@@ -6,16 +6,26 @@ runs:
   steps:
     - uses: actions/setup-node@0a44ba7841725637a19e28fa30b79a866c81b0a6 # v4.0.4
       with:
-        node-version: 20
-        cache: <%= packageManager %>
+        node-version: 20<% if (packageManager !== 'yarn') { %>
+        cache: <%= packageManager %><% } %>
     - name: Enable Corepack
       shell: bash
-      run: corepack enable
-    - name: Install
-<% if (npmRegistry) { %>
+      run: corepack enable<% if (packageManager === 'yarn') { %>
+    - name: Get yarn cache directory path
+      shell: bash
+      id: yarn-cache-dir-path
+      run: echo "dir=$(<%= yarnCacheFolder %>)" >> $GITHUB_OUTPUT
+    - name: Cache dependencies
+      uses: actions/cache@6849a6489940f00c2f30c0fb92c6274307ccb58a # v4.1.2
+      with:
+        path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
```

Comment:
This is not missing the following files:
- .yarn/unplugged
- .pnp.cjs
- .pnp.loader.mjs
?

Reply:
not sure those are needed. I followed the recommendations from https://github.com/actions/cache/blob/main/examples.md#node---yarn